tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -76,50 +86,49 @@ def test_rb_command_with_force_deletes_objects_in_bucket(self):
-                .call.return_value = (None, {'Buckets': []})
-        self.session.get_service.return_value.get_operation.return_value\
+        self.session.create_client.return_value.list_buckets.return_value\
+            = {'Buckets': []}
+        self.session.create_client.return_value.get_paginator.return_value\
                 .paginate.return_value = [
-                    (None, {'Contents': [], 'CommonPrefixes': []})]
+                    {'Contents': [], 'CommonPrefixes': []}]
 
     def test_ls_command_for_bucket(self):
         ls_command = ListCommand(self.session)
-        parsed_args = FakeArgs(paths='s3://mybucket/', dir_op=False, page_size='5',
-                human_readable=False, summarize=False)
+        parsed_args = FakeArgs(paths='s3://mybucket/', dir_op=False,
+                               page_size='5', human_readable=False,
+                               summarize=False)
         parsed_globals = mock.Mock()
         ls_command._run_main(parsed_args, parsed_globals)
-        call = self.session.get_service.return_value.get_operation\
-                .return_value.call
-        paginate = self.session.get_service.return_value.get_operation\
+        call = self.session.create_client.return_value.list_objects
+        paginate = self.session.create_client.return_value.get_paginator\
                 .return_value.paginate
         # We should make no operation calls.
         self.assertEqual(call.call_count, 0)
         # And only a single pagination call to ListObjects.
-        self.session.get_service.return_value.get_operation.assert_called_with(
-            'ListObjects')
-        self.assertEqual(
-            paginate.call_args[1], {'bucket': u'mybucket',
-                                    'delimiter': '/', 'prefix': u'',
-                                    'page_size': u'5'})
+        self.session.create_client.return_value.get_paginator.\
+            assert_called_with('list_objects')
+        ref_call_args = {'Bucket': u'mybucket', 'Delimiter': '/',
+                         'Prefix': u'', 'page_size': u'5'}
+
+        paginate.assert_called_with(**ref_call_args)
 
     def test_ls_command_with_no_args(self):
         ls_command = ListCommand(self.session)
-        parsed_global = FakeArgs(region=None, endpoint_url=None, verify_ssl=None)
+        parsed_global = FakeArgs(region=None, endpoint_url=None,
+                                 verify_ssl=None)
         parsed_args = FakeArgs(dir_op=False, paths='s3://',
                                human_readable=False, summarize=False)
         ls_command._run_main(parsed_args, parsed_global)
         # We should only be a single call.
-        self.session.get_service.return_value.get_operation.assert_called_with(
-            'ListBuckets')
-        call = self.session.get_service.return_value.get_operation\
-                .return_value.call
+        call = self.session.create_client.return_value.list_buckets
+        self.assertTrue(call.called)
```

Comment:
This will be asserted by the `call.called_count` check anyway, so you don't really need to assert `call.called`.